themes/default/content/registry/packages/eventstorecloud/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Event Store Cloud
 meta_desc: Learn how you can use Event Store Cloud Provider for Pulumi to provision and manage Event Store Cloud resources.
-layout: package
+layout: overview
```

Comment:
@cnunciato the upstream content still seems to have these obsolete values.

Reply:
@ringods Yeah, this is fine -- not ideal, but the build supports both. (I made this work in https://github.com/pulumi/registry/pull/2797.)